docs/3.0.0/docs/daml-finance/index.rst

```diff
@@ -76,7 +75,98 @@ last release.
+   implement. This new standard has guided the renaming and structuring of holding implementations.
+   The ``Fungible`` interface no longer requires the ``Transferable`` interface. However, both
+   ``Transferable`` and ``Fungible`` continue to require the implementation of the ``Holding``
+   interface (renamed from ``Base`` following customer feedback). Moreover, the settlement process
+   has been refined to require only a matching ``HoldingStandard``, allowing for implementation
+   variations.
+
+#. A unified ``HoldingFactory`` capable of creating holdings for any specified ``HoldingStandard``
+   has been adopted. In particular, this enables multiple holdings (of various ``HoldingStandards``)
+   to be credited to the same account.
+
+Foreseeing future integration with Daml 3.0 and the Canton Network
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to ease future transitions to Daml 3.0 and the Canton Network we have shifted to
+single-maintainer contract keys:
+
+#. The `issuer : Party` of the ``InstrumentKey`` is now the single maintainer for the ``Instrument``
+   key.
+
+#. For ``Batch`` and ``Instruction``, the `requestors : Parties` field has been divided into a
+   single-maintainer `instructor : Party` for the ``Instruction`` key, alongside additional
+   signatories `consenters : Parties`. Corresponding changes have been made to the ``Batch`` and
+   ``Instruction`` views. In the ``Daml.Finance.Lifecycle.Rule.Claim`` implementation,
+   `providers : Parties` has been replaced with a single `provider : Party` (to facilitate assigning
+   the `provider` as a settlemetn `instructor : Party`).
+
+#. The LedgerTime key has been completely removed as it was redundant.
+
+Streamlining Interface Archival
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Previously, our factory contracts featured a ``Remove`` choice for archiving interface instances.
+With Daml now supporting direct archival of interface instances, these choices have been removed. To
+facilitate the simultaneous archival of ``Account``, ``Instrument``, and ``HoldingFactory``
+interfaces with their related ``Reference`` contract instance, a ``Remove`` choice has been added to
+the ``Account``, base ``Instrument``, and ``HoldingFactory`` interfaces.
+
+New Interface Lockable
+~~~~~~~~~~~~~~~~~~~~~~
+
+The locking mechanism has been separated from the base ``Holding`` interface into a new ``Lockable``
+interface (which the ``Holding`` now requires). That makes ``Lockable`` available for broader use;
+while the ``Account`` also implements ``Lockable`` allowing to freeze an account, it’s not required.
+
+Additionally, the implementations of ``Transfer``, ``Split``, ``Merge``, and ``Debit`` have been
+adjusted to require unlocking before they can be used when in the locked state.
+
+New Instruments
+~~~~~~~~~~~~~~~
+
+Furthermore, we have broadened the library's functionality by introducing additional packages for
```

Comment:
```suggestion
Furthermore, we have broadened the library's functionality by introducing
```

Reply:
Only the structured product is in a new package, the new basket asset swap is in an existing package. I would maybe not mention package at all, but just focus on the new functionality offered in this release.